apps/docs/docs/template/api/rx-for-directive.md

```diff
@@ -49,6 +49,7 @@ Each instance of `RxFor` can be configured to render with different settings.
 - Provide a comprehensive set of context variables
 
 **Performance Features**
+-optimizations
```

Comment:
same here

Reply:
good catch!
damn maybe some wild GitHub Copilot here :D 
will fix this of course